web/client/codechecker_client/client.py

```diff
@@ -58,13 +58,11 @@ def setup_auth_client(protocol, host, port, session_token=None):
 
 def handle_auth(protocol, host, port, username, login=False):
```

Comment:
I think we can rename this function because we use it only for login and handle_auth does not say a lot of information.
`login_user` or something like this would be better, please add some function documentation (if login is false the user will be logged out).